ruby-training/trainees/shareni/test_katas/05/sum_array/new_array.rb

```diff
@@ -0,0 +1,13 @@
+class NewArray
+    def initialize(example_array)
+        @example_array = example_array
+    end
+
+    def sum_without_max_and_min
+        array_without_max_and_min.reduce(:+)
+    end
+
+    def array_without_max_and_min
```

Comment:
Even though this method doesn't seem to be developer facing (it's more of an intermediate method), you should still test it to make sure your code is covered. Someone could, in the future, come and change this method, break it, and not notice since there are no specs to tell them it's not working.